### PR TITLE
[FIX] payment_adyen, payment_authorize, payment_buckaroo, payment_pay…

### PR DESCRIPTION
**Pull request by simongoffin**
…pal, payment_sips: Disable csrf on callbacks

Inspired from 2099f15d67564ab153d48993cca0f80aee04bb09

opw:653341

### CODE REVIEW

File: addons/payment_adyen/controllers/main.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def adyen_return(self, **post):
 
     @http.route([
         '/payment/adyen/notification',
-    ], type='http', auth='none', methods=['POST'])
+    ], type='http', auth='none', methods=['POST'], csrf=False)
```
Comment (mart-e) [inline addons/payment_adyen/controllers/main.py]: where is this one called from?

File: addons/payment_ogone/controllers/main.py [hunk 2]
```diff
@@ -21,7 +21,7 @@ class OgoneController(http.Controller):
         '/payment/ogone/decline', '/payment/ogone/test/decline',
         '/payment/ogone/exception', '/payment/ogone/test/exception',
         '/payment/ogone/cancel', '/payment/ogone/test/cancel',
-    ], type='http', auth='none')
+    ], type='http', auth='none', csrf=False)
     def ogone_form_feedback(self, **post):
```
Comment (mart-e) [inline addons/payment_ogone/controllers/main.py]: is that a POST request?

**Comment by mart-e** [comment]
You should put CSRF only on POST requests coming from an external server. Are you sure it's the case for all those routes?

File: addons/payment_adyen/controllers/main.py (hunk 1 above)
Comment (nle-odoo) [inline addons/payment_adyen/controllers/main.py]: looking at https://github.com/odoo/odoo/commit/3dafb19c96 and https://docs.adyen.com/display/TD/Set+up+notifications it has to be set manually on adyen

Comment (mart-e) [inline addons/payment_adyen/controllers/main.py]: @nle-odoo indeed from adyen so ok for this csrf

**Pull request merged**